modules/ROOT/pages/design-named-examples.adoc

```diff
@@ -208,9 +208,28 @@ types:
     properties:
       givenName: string
       familyName: string
-   example: !include fragment1.raml
+   example: !include fragment1.yaml
 ----
 
+.fragment1.yaml
+----
+givenName: ”Chiaki”
+familyName: "Mukai"
+----
+
+.fragment2.yaml
+----
+givenName: "Kyung-won"
+familyName: "Park"
+----
+
+
+[NOTE]
+====
+According to the RAML 1.0 specification, a NamedExample fragment that defines only one example is not valid. Therefore, you should not use NamedExample fragments when you are using single, unnamed examples.
```

Comment:
Is this ref. to RAML still valid, given the file name extension change? Just double-checking....

Reply:
Yes. I've revised the beginning of the note to make more clear that the note explains why the example uses .yaml files and not .raml files.